closure/goog/demos/keyhandler.html

```diff
@@ -98,6 +98,20 @@
   }
 
   switch (e.keyCode) {
+
+  // Add the keycodes for the number-pad keys
+
+	case goog.events.KeyCodes.NUM_ZERO:
```

Comment:
I apologize. Yes - my mistakes. I would fix these myself but I don't want to cause any problems. If you / whomever could fix this, I would be grateful.

Thank you!